sunpy/timeseries/timeseriesbase.py

```diff
@@ -243,8 +190,228 @@ def time_range(self):
         else:
             return None
 
+    @property
+    def url(self):
+        """
+        URL to the mission website.
+        """
+        return self._url
+
 # #### Data Access, Selection and Organisation Methods #### #
 
+    def _text_summary(self):
+        """
+        Produces a table summary of the timeseries data to be passed to
+        the _repr_html_ function.
+        """
+        obs = self.observatory
+        if obs is None:
+            try:
+                obs = self.meta.metadata[0][2]["telescop"]
+            except KeyError:
+                obs = "Unknown"
+        try:
+            inst = self.meta.metadata[0][2]["instrume"]
+        except KeyError:
+            inst = "Unknown"
+        try:
+            link = f"""<a href={self.url} target="_blank">{inst}</a>"""
+        except AttributeError:
+            link = inst
+        dat = self.to_dataframe()
+        drange = dat.max() - dat.min()
+        drange = drange.to_string(float_format="{:.2E}".format)
+        drange = drange.replace("\n", "<br>")
+
+        center = self.time_range.center.value.astype('datetime64[s]')
+        center = str(center).replace("T"," ")
+        resolution = round(self.time_range.seconds.value/self.shape[0],3)
+        resolution = str(resolution)+" s"
+
+        channels = self.columns
+        channels = "<br>".join(channels)
+
+        uni = list(set(self.units.values()))
+        uni = [x.unit if type(x) == u.quantity.Quantity else x for x in uni]
+        uni = ["dimensionless" if x == u.dimensionless_unscaled else x for x in uni]
+        uni = "<br>".join(str(x) for x in uni)
+
+        return textwrap.dedent(f"""\
+                   SunPy TimeSeries
+                   ----------------
+                   Observatory:\t\t {obs}
+                   Instrument:\t\t {link}
+                   Channel(s):\t\t {channels}
+                   Start Date:\t\t {dat.index.min().round('s')}
+                   End Date:\t\t {dat.index.max().round('s')}
+                   Center Date:\t\t {center}
+                   Resolution:\t\t {resolution}
+                   Samples per Channel:\t\t {self.shape[0]}
+                   Data Range(s):\t\t {drange}
+                   Units:\t\t {uni}\
```

Comment:
What is the last `\` for?

Reply:
Nothing in particular :). I will remove that in an update to this PR.